vms/components/multisig/camino_multisig.go

```diff
@@ -4,13 +4,42 @@
 package multisig
 
 import (
+	"fmt"
+
 	"github.com/ava-labs/avalanchego/ids"
+	"github.com/ava-labs/avalanchego/snow"
+	"github.com/ava-labs/avalanchego/utils/hashing"
 	"github.com/ava-labs/avalanchego/vms/components/verify"
 	"github.com/ava-labs/avalanchego/vms/types"
 )
 
+// we can't use avax.MaxMemoSize because for some reason it results in import cycle even if it isn't
+
```

Comment:
I think the import cycle exists:
-> multisig import avax (If we add avax.MaxMemoSize)
-> avax imports secp256k1fx
-> secp256k1fx imports multisig in camino_fx

but IMO for the time being we should just remove this comment instead of trying to resolve the import cycle just for this const.

Reply:
You are right, this looks like the reason
And its actually bad - only avax (and one of ours) test files import secp256k1fx - its not intended by implementation. That ruins package usage a bit.

Reply:
Yep, true.. But still avax code that we shouldn't touch :) So we can just redefine the memoSize as you did.